ui/apps/platform/src/Containers/Vulnerabilities/VulnerablityReporting/VulnReports/VulnReportingLayout.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { PageSection, Title, Tabs, Tab } from '@patternfly/react-core';
+import { Outlet, useLocation, useNavigate } from 'react-router-dom';
+
+import useFeatureFlags from 'hooks/useFeatureFlags';
+import PageTitle from 'Components/PageTitle';
+import {
+    vulnerabilityConfigurationReportsPath,
+    vulnerabilityOnDemandReportsPath,
+} from 'routePaths';
+
+const tabs = [
+    {
+        id: 'report-configuration',
+        title: 'Report configurations',
+        path: vulnerabilityConfigurationReportsPath,
+    },
+    { id: 'on-demand-reports', title: 'On-demand reports', path: vulnerabilityOnDemandReportsPath },
+];
+
+function VulnReportingLayout() {
+    const { isFeatureFlagEnabled } = useFeatureFlags();
+    const isOnDemandReportsEnabled = isFeatureFlagEnabled('ROX_VULNERABILITY_ON_DEMAND_REPORTS');
+
+    const location = useLocation();
+    const navigate = useNavigate();
+
+    const activeTabIndex = tabs.findIndex((tab) => location.pathname.startsWith(tab.path));
```

Comment:
Using the path to map to the active tab is something I don't think I've seen before in the app, but if we are using path based URLs for tabs this is a great way to do this consistently. 👍